interactive_engine/executor/ir/proto/algebra.proto

```diff
@@ -469,6 +469,9 @@ message SinkVineyard {
   schema.Schema graph_schema = 2;
 }
 
+// A dummy node to delegate a source opr for multiple scan cases.
+message RootScan {}
```

Comment:
Do not like `RootScan`, Simply call it `Root`.

Reply:
Done.